code/modules/mob/abstract/new_player/preferences_setup.dm

```diff
@@ -238,18 +238,22 @@ datum/preferences
 	dress_preview_mob(mannequin)
 
 	preview_icon = icon('icons/effects/effects.dmi', "nothing")
-	preview_icon.Scale(48+32, 16+32)
+	preview_icon.Scale(32*3, 32)

```

Comment:
What are the changes in this file?

Reply:
Oh some things I forgot to get rid of however it may make a nice addition. It basically makes it so the preview screen is less scattered. The profile is now lined up nicely.

Reply:
Probably should be split into a separate PR.